jwala-tomcat/src/main/resources/data/scripts/delete-service.sh

```diff
@@ -22,9 +22,11 @@ fi
 
 if $linux; then
```

Comment:
Can you remove one if check?

Reply:
Sure, I guess we just wanted to make sure it was really Linux :)